mindmeld/components/request.py

```diff
@@ -295,6 +295,10 @@ class FrozenParams(Params):
     dynamic_resource = attr.ib(default=immutables.Map(), converter=immutables.Map)
 
 
+def list_elems_to_immutable_map(value):
```

Comment:
what is the type/structure for value?

Reply:
value represents the object that is being passed in for the given attribute so its type is the same as the type of the object

Reply:
Isnt the value supposed to be just a dictionary? For instance `immutables.Map(i)` cannot be done if `i = "cat"`, no?

Reply:
Yes, in this case it would be a tuple containing one or more dictionaries as that is the type/structure we are expecting for the respective Request attributes. It might be helpful if I change the name of this custom attrs converter to "tuple_elems_to_immutable_map" as it expects tuples not lists.

Reply:
It would also be good to check if value is a list of maps before converting it as well.